### ISSUE TITLE
Curtain (Window Covering) 2CH device showing only first curtain

### ISSUE THREAD

**Issue by CryptoIR** [0x5e/homebridge-tuya-platform#325]
### Prerequisite

- [X] Yes, I've read the readme completely.

### Cache

- [X] Yes, I've cleaned accessory cache and the issue still exists.

### Version

@0x5e/homebridge-tuya-platform v1.7.0-beta.45

### Device Infomation JSON File

```json
"status": [
      {
        "code": "control",
        "value": "stop"
      },
      {
        "code": "control_2",
        "value": "stop"
      },
      {
        "code": "control_back_mode",
        "value": "forward"
      },
      {
        "code": "control_back_mode_2",
        "value": "forward"
      },
      {
        "code": "cur_calibration",
        "value": "end"
      },
      {
        "code": "cur_calibration_2",
        "value": "end"
      },
      {
        "code": "percent_control",
        "value": 100
      },
      {
        "code": "percent_control_2",
        "value": 100
      }
    ],


    "schema": [
      {
        "code": "control",
        "mode": "rw",
        "type": "Enum",
        "property": {
          "range": [
            "open",
            "stop",
            "close"
          ]
        }
      },
      {
        "code": "control_2",
        "mode": "rw",
        "type": "Enum",
        "property": {
          "range": [
            "open",
            "stop",
            "close"
          ]
        }
      },
      {
        "code": "control_back_mode",
        "mode": "rw",
        "type": "Enum",
        "property": {
          "range": [
            "forward",
            "back"
          ]
        }
      },
      {
        "code": "control_back_mode_2",
        "mode": "rw",
        "type": "Enum",
        "property": {
          "range": [
            "forward",
            "back"
          ]
        }
      },
      {
        "code": "cur_calibration",
        "mode": "rw",
        "type": "Enum",
        "property": {
          "range": [
            "start",
            "end"
          ]
        }
      },
      {
        "code": "cur_calibration_2",
        "mode": "rw",
        "type": "Enum",
        "property": {
          "range": [
            "start",
            "end"
          ]
        }
      },
      {
        "code": "percent_control",
        "mode": "rw",
        "type": "Integer",
        "property": {
          "unit": "%",
          "min": 0,
          "max": 100,
          "scale": 0,
          "step": 10
        }
      },
      {
        "code": "percent_control_2",
        "mode": "rw",
        "type": "Integer",
        "property": {
          "unit": "%",
          "min": 0,
          "max": 100,
          "scale": 0,
          "step": 10
        }
      }
    ]
```

### Device Control Mode

Standard Instruction

### Other Infomations

2CH Zigbee Curtain Module

I expect to see 2 items as in 2CH switches

**Comment by CryptoIR** [comment]
@0x5e Can I get permission to open a branch and PR?

**Comment by 0x5e** [comment]
@CryptoIR sorry I don't have too much time to commit recently, you can fork the project and pr, I will merge it when I'm free. thanks in advance.

**Comment by CryptoIR** [comment]
PR #339 @0x5e opened

**Comment by 0x5e** [comment]
Fixed in #339
